templates/agent-conf.d/http_check.yaml.erb

```diff
@@ -28,7 +28,11 @@ instances:
 <% if instance['collect_response_time'] -%>
     collect_response_time: <%= instance['collect_response_time'] %>
 <% end -%>
+<% if instance['disable_ssl_validation'] -%>
     disable_ssl_validation: <%= instance['disable_ssl_validation'] %>
+<% else %>
+    disable_ssl_validation: false
```

Comment:
We should just remove this.... the if [here in line 31](https://github.com/DataDog/puppet-datadog-agent/pull/258/files#diff-cccc5b6ffc3ddf9109ba688b10b6ce97R31) should be enough... because the default behavior for the check is for disable_ssl_validation to be `true`.